src/components/common/Confirm.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Text, View, Modal } from 'react-native';
+import { CardSection } from './CardSection';
+import { Button } from './Button';
+
+const Confirm = ({ children, visible, onAccept, onDecline }) => {
+  const { containerStyle, textStyle, CardSectionStyle } = styles;
+
+  return (
+    <Modal
+      visible={visible}
+      transparent
+      animationType="slide"
+      onRequestClose={() => {}}
+    >
```

Comment:
The broker asks you to do this, look:
![image](https://cloud.githubusercontent.com/assets/17891668/26533468/bf7f9956-43ef-11e7-9051-a51a99f496d9.png)